packages/commonwealth/server/migrations/20250409185523-create-nomination-groups.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const GROUP_NAME = 'Community Nominated';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    const now = new Date();
+
+    // get communities with chain node and namespace
+    const communities = await queryInterface.sequelize.query(
+      `SELECT c.id, c.namespace_address, cn.eth_chain_id
+       FROM "Communities" c
+       JOIN "ChainNodes" cn ON c.chain_node_id = cn.id
+       WHERE c.namespace_address IS NOT NULL`,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    if (!communities.length) {
+      console.warn(
+        'No communities found with a valid namespace_address and ChainNode.',
+      );
+      return;
+    }
+
+    // prepare groups and bulk insert
+    const groups = communities.map((community) => ({
+      community_id: community.id,
```

Comment:
Does nominated users apply even for legacy namespaces?

Reply:
it should yea can be enabled